.prettierrc

```diff
@@ -2,5 +2,6 @@
   "bracketSpacing": false,
   "singleQuote": true,
   "trailingComma": "es5",
-  "quoteProps": "preserve"
+  "quoteProps": "preserve",
+  "arrowParens": "avoid"
```

Comment:
Without this, prettier forces parens around all arrow functions, which is a new rule. Disabling this for now (can have discussion in separate thread if this is controversial) so this can just update packages.

Reply:
👍